Coverage reporter shallow error doc (#52746)
nodejs/node: doc/api/test.md

node --test --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=lcov.info
```

* No test results are reported by this reporter.

> * This reporter should ideally be used in conjunction with another reporter.

Reviewer: ```suggestion
* This reporter should ideally be used alongside another reporter.
```

"in conjunction" is probably less easy to understand for ESL folk
